MolecularNodes/md.py

```diff
@@ -107,10 +160,37 @@ def load_trajectory(file_top,
     
     
     # determin the bonds for the structure
+    # if hasattr(univ, 'bonds') and include_bonds:
+    #     bonds = univ.bonds.indices
+    # else:
+    #     bonds = []
+
+
+
     if hasattr(univ, 'bonds') and include_bonds:
-        bonds = univ.bonds.indices
+    # If there is a selection, we need to recalculate the bond indices
```

Comment:
This is the only section that should be in the PR you open. You should create a branch from the main branch on the MolecularNodes repo and just add this. This is the only part of the code that addresses the bug.

Reply:
Ok, so delete this? I remember it conflicted with the selection feature which is why I decided to keep this bug fix, so it doesn't crash. I'm implementing something similar to "selection" except it has 6 different selection inputs.